### PR TITLE
Adds the ED-CLN

### PR DESCRIPTION
**Pull request by MisterLayne**
What the title says. Better version of the cleanbot.
To do:

- [x] Figure out why the href shit isn't working at all, actually.

### CODE REVIEW

File: code/modules/mob/living/bot/edCLNbot.dm [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+/mob/living/bot/cleanbot/edCLN/update_icons()
+	if(on && busy)
+		icon_state = "edCLN"
+	else
+		icon_state = "edCLN[on]"
+
+/mob/living/bot/cleanbot/edCLN/handleIdle()
+	if(prob(10))
+		custom_emote(2, "makes a less than thrilled beeping sound.")
+		playsound(src.loc, 'sound/machines/synth_yes.ogg', 50, 0)
```
Comment (Belsima) [inline code/modules/mob/living/bot/edCLNbot.dm]: beloved

Comment (MisterLayne) [reply code/modules/mob/living/bot/edCLNbot.dm]: wut

**Comment by MisterLayne** [comment]
@Anewbe tested and finished.

File: code/modules/mob/living/bot/edCLNbot.dm (hunk 1 above)
Comment (Belsima) [reply code/modules/mob/living/bot/edCLNbot.dm]: Beloved.

**Pull request merged**